### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by bhuvy2** [illinois-cs241/coursebook#1]
So the website needs to be accessible to those with screen readers. Latex and PDFs are not the best format to do this, but HTML is arguably leagues better. Since Latex is just a markup language, we should be able to convert between this one and other ones. We can use pandoc to convert the latex into flat HTML and then transform it into accessible html that we then push to the website.

https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML

**Comment by bhuvy2** [comment]
We currently (almost) have wiki accessibility. We just need to get captions for images.

**Comment by bhuvy2** [comment]
Captions has been gotten for images. We now have almost 100% accessibility.
